src/structures/ClientApplication.js

```diff
@@ -23,6 +30,27 @@ class ClientApplication extends Application {
   _patch(data) {
     super._patch(data);
 
+    this.tags = data.tags ?? [];
```

Comment:
This isn't documented.

Reply:
What do you mean? It got documented in discord/discord-api-docs#4112 right?

Reply:
Correct. However, it has no JSDoc here. Without JSDoc, it will not appear in discord.js's documentation. Look at the implementation in #7000 for example:

https://github.com/discordjs/discord.js/blob/0673d38afe63b6f46783f3f7dd7bbe17a0a015cf/packages/discord.js/src/structures/ClientApplication.js#L33-L37

The first 4 lines are the JSDoc that describes the property, but that is not here. This will result in the property not showing in our documentation.